ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -170,6 +171,16 @@ gcp_connection::object_exists(const std::string &object_key, bool &exists, size_
     }
 
     std::cerr << object_key + ": " + metadata.status().message() << std::endl;
+    return get_errno(metadata.status());
+}
 
+// Maps Google Cloud Status code to an Errno value which is returned.
+// If a mapping cannot be found the Google Cloud Status code is printed and -1 is returned.
+int
+gcp_connection::get_errno(const google::cloud::Status status) const
+{
+    if (toErrno.find(status.code()) != toErrno.end())
+        return (toErrno.at(status.code()));
+    std::cerr << status.code() << std::endl;
```

Comment:
What is the output of `status.code()`? Are we just printing a single number with no context?

Reply:
The status.code() is an enum defined here: https://googleapis.dev/cpp/google-cloud-common/2.6.0/status_8h_source.html

For example "UNKNOWN" could be printed out here.

The status message has more information which is printed out in the connection class functions if there is an error.

Reply:
Ok, I'm still confused as to why we are printing it out without context as it doesn't seem like useful information for the user. Do we need to print anything at all?

Reply:
Hi Tammy, 
I thought terminating with an error code would be better than not printing out any error messages which could appear to be successful.
I am not sure of a better way to handle this scenario, any suggestions are welcome :)

Reply:
Is `status.code()` an integer? If so, why not return that value instead of `-1`?

Reply:
`status.code()` is an enum class

Reply:
@sonar3ddy enum's are just integers under the hood - for example in the linked source code from above we have `kAlreadyExists = 6` which means if we returned `status.code()` here we'd end up returning the integer 6. 

That said I prefer returning `-1` here. The goal of this ticket - taken from the history of WT-10368 - is `This ticket will also map the google cloud errors to WT local errors. This means that we will read the google cloud potential errors and then change them to fit the WT language muchalike S3's implementation.` indicating we shouldn't return any GCP error codes. We do the same thing in the S3 code where we fall back on `-1` when the `toErrno.find()` call fails

Reply:
@sonar3ddy will you please remove the `std::cerr << status.code() << std::endl;` on line 184? Thank you.